packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -22,85 +22,19 @@ import Button, { ButtonProps } from '../Button';
 type Action = Omit<ButtonProps, 'variant'>;
 
 export interface ButtonGroupProps {
-  /**
-   * @deprecated Use the `actions` prop instead.
-   */
-  children?:
-    | (ReactElement<ButtonProps> | null | undefined)[]
-    | ReactElement<ButtonProps>;
   /**
    * Direction to align the content. Either left/right
    */
   align?: 'left' | 'center' | 'right';
```

Comment:
Changing the alignment is no longer supported, so this prop should be removed.

However, when trying to migrate to the new `actions` API in the web dashboard, I found many use cases where the default center alignment doesn't work. That's why I think we need to reconsider whether we can drop the alignment option.